excercise9/itemmaster/ItemMasterDTO.java

```diff
@@ -0,0 +1,96 @@
+package itemmaster;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class ItemMasterDTO implements Serializable,Cloneable,Comparable<ItemMasterDTO>{
+	private int item_id;
+	private String item_name;
```

Comment:
As per Java standards, variable names should follow the camel-casing format. Can you please update the variable names?

Reply:
All the variable names changed to camel-case format